packages/jsii/lib/assembler.ts

```diff
@@ -487,7 +546,7 @@ export class Assembler implements Emitter {
   ): Promise<void> {
     const declaration = symbol.valueDeclaration ?? symbol.declarations[0];
     if (declaration == null) {
-      // Nothing to do here...
+      // Nothing to do here..
```

Comment:
:/  I vote either zero, one, or three; two periods/full stops is just weird.
```suggestion
      // Nothing to do here
```